component/attr_cache/attr_cache.go

```diff
@@ -893,9 +876,14 @@ func (ac *AttrCache) CreateLink(options internal.CreateLinkOptions) error {
 	if err == nil {
 		ac.cacheLock.RLock()
 		defer ac.cacheLock.RUnlock()
-		ac.invalidatePath(options.Name)
-		if ac.cacheDirs {
-			ac.markAncestorsInCloud(getParentDir(options.Name), time.Now())
+		toBeInvalid, err := ac.cacheMap.get(options.Name)
+		if err != nil {
+			log.Err("AttrCache::CreateLink : The attribute item could not be invalidated in the cache due to the following error: ", err)
+		} else {
+			toBeInvalid.invalidate()
+			if ac.cacheDirs {
+				ac.markAncestorsInCloud(getParentDir(options.Name), time.Now())
+			}
 		}
```

Comment:
```suggestion
		toBeInvalid, getErr := ac.cacheMap.get(options.Name)
		if getErr == nil {
			toBeInvalid.invalidate()
        }
        if ac.cacheDirs {
            ac.markAncestorsInCloud(getParentDir(options.Name), time.Now())
        }
```
1. No need to log an error here.
2. Only the invalidation should depend on whether get() succeeded or not. We should remember that the link's parent directory is in the cloud regardless.
3. If get() returns an error, it should not overwrite the nil err returned by `NextComponent().CreateLink()`.